hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -792,20 +755,19 @@ public List<BackupInfo> getBackupHistory(int n, BackupInfo.Filter... filters) th
     return result;
   }
 
-  /*
-   * Retrieve TableName's for completed backup of given type
-   * @param type backup type
-   * @return List of table names
+  /**
+   * Retrieve all table names that are part of any known backup
    */
-  public List<TableName> getTablesForBackupType(BackupType type) throws IOException {
+  public Set<TableName> getTablesForBackupType() throws IOException {
```

Comment:
This method should probably be renamed.

Reply:
Yeah, getFullyBackedUpTables or getTablesWithFullBackups would be improve the clarity